tools/rtrclient.c

```diff
@@ -106,7 +106,7 @@ static void update_spki(struct spki_table* s __attribute__((unused)), const stru
 
 int main(int argc, char** argv)
 {
-    enum mode_t { TCP, SSH } mode;
+    enum mode_t { NOP, TCP, SSH } mode = NOP;
```

Comment:
Yes it does fix such warning. Still, I would favor a distinct 'not set' state, e.g. _NOP_ or something similar.